src/data/actions/grades.js

```diff
@@ -72,7 +73,7 @@ const fetchGrades = (courseId, cohort, track, showSuccess) => (
           data.results.sort(sortAlphaAsc),
           cohort,
           track,
-          headingMapper.all(dispatch, data.results[0]),
+          headingMapper('All-Assignments')(dispatch, data.results[0]),
```

Comment:
Maybe move this label text to a single const variable? Please update this to `All`